src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -1284,6 +1284,19 @@ def _validate_vm_vmss_msi(cmd, namespace, is_identity_assign=False):
     elif namespace.identity_scope or namespace.identity_role:
         raise ArgumentUsageError('usage error: --assign-identity [--scope SCOPE] [--role ROLE]')
 
+    # Enable system assigned msi by default when Trusted Launch configuration is met
+    if hasattr(namespace, 'disable_integrity_monitoring') and not namespace.disable_integrity_monitoring:
+        if hasattr(namespace, 'security_type') and hasattr(namespace, 'enable_vtpm') \
+                and hasattr(namespace, 'enable_secure_boot'):
+            if namespace.security_type and namespace.security_type.lower() == 'trustedlaunch' and\
+                    namespace.enable_vtpm and namespace.enable_secure_boot:
```

Comment:
Please use a variable `is_trusted_launch` to abstract these complex judgments

Reply:
It seems that `security_type` `enable_vtpm` and `enable_secure_boot` cannot be included with `is_trusted_launch`